game-core/src/main/java/games/strategy/triplea/ui/menubar/GameMenu.java

```diff
@@ -228,60 +229,32 @@ private void addRollDice() {
           panel.setLayout(new GridBagLayout());
           panel.add(
               new JLabel("Number of Dice to Roll: "),
-              new GridBagConstraints(
-                  0,
-                  0,
-                  1,
-                  1,
-                  0,
-                  0,
-                  GridBagConstraints.WEST,
-                  GridBagConstraints.BOTH,
-                  new Insets(0, 0, 0, 20),
-                  0,
-                  0));
+              new GridBagConstraintsBuilder(0, 0)
+                  .anchor(GridBagConstraintsAnchor.WEST)
+                  .fill(GridBagConstraintsFill.BOTH)
+                  .insets(0, 0, 0, 20)
+                  .build());
           panel.add(
               new JLabel("Sides on the Dice: "),
-              new GridBagConstraints(
-                  2,
-                  0,
-                  1,
-                  1,
-                  0,
-                  0,
-                  GridBagConstraints.WEST,
-                  GridBagConstraints.BOTH,
-                  new Insets(0, 20, 0, 10),
-                  0,
-                  0));
+              new GridBagConstraintsBuilder(2, 0)
+                  .anchor(GridBagConstraintsAnchor.WEST)
+                  .fill(GridBagConstraintsFill.BOTH)
+                  .insets(0, 20, 0, 10)
+                  .build());
           panel.add(
               numberOfText,
-              new GridBagConstraints(
-                  0,
-                  1,
-                  1,
-                  1,
-                  0,
-                  0,
-                  GridBagConstraints.WEST,
-                  GridBagConstraints.BOTH,
-                  new Insets(0, 0, 0, 20),
-                  0,
-                  0));
+              new GridBagConstraintsBuilder(0, 1)
+                  .anchor(GridBagConstraintsAnchor.WEST)
+                  .fill(GridBagConstraintsFill.BOTH)
+                  .insets(0, 0, 0, 20)
+                  .build());
           panel.add(
               diceSidesText,
-              new GridBagConstraints(
-                  2,
-                  1,
-                  1,
-                  1,
-                  0,
-                  0,
-                  GridBagConstraints.WEST,
-                  GridBagConstraints.BOTH,
-                  new Insets(0, 20, 0, 10),
-                  0,
-                  0));
+              new GridBagConstraintsBuilder(2, 1)
+                  .anchor(GridBagConstraintsAnchor.WEST)
+                  .fill(GridBagConstraintsFill.BOTH)
+                  .insets(0, 20, 0, 10)
+                  .build());
```

Comment:
Sometimes I really ask myself why the designers of swing didn't make use of such a builder pattern.
It's so much more readable 👍